src/main/scala/CurlRequests/Agents.scala

```diff
@@ -14,14 +14,26 @@
  * limitations under the License.
  */
 
-package controllers
+package CurlRequests
```

Comment:
don't use upper case letters in package names

http://docs.scala-lang.org/style/naming-conventions.html#packages